### DISCUSSION TITLE
The "Status Checks" Section of Branch Rules Needs Better Clarity For Actions

### DISCUSSION THREAD

**Discussion by mattlavinder** [community/community#53824] [Actions]
### Select Topic Area

Product Feedback

### Body

I was trying to figure out how to require an action to be successful before merging could occur today and it took much longer than it should have to figure out.  The section on "status checks" under branch rules only has links to documentation about the API used for creating status checks, and nothing is said anywhere about Github actions.

I eventually figured out my job names were "status checks" I could include here, but I just got lucky and guessed that. I am sure this is all covered somewhere. In fact, I probably read about this at some point and didn't know what it was talking about, so I disregarded it.  Still, this is so basic, I should not have needed to spend more than 1 minute of my day on this. I am sure everyone wants to do such a basic thing, and it is super easy to make it clear. 

The header of the table could say, "Status checks or action jobs that are required"
If you want to be more technical about it, add a short sentence somewhere saying, "Github action job names may be used as status checks."

**Comment by patrick-knight** [comment]
Thanks for the feedback @mattlavinder!
